profiles/kde.py

```diff
@@ -20,6 +20,17 @@ def _prep_function(*args, **kwargs):
 		else:
 			print('Deprecated (??): xorg profile has no _prep_function() anymore')
 
+def _post_install(*args, **kwargs):
+	if "nvidia" in _gfx_driver_packages:
+		print("Plasma wayland is currently in a buggy state on Nvidia cards")
+	choice = input("Kde plasma has a wayland support would you like to install the required binaries [Y/n] ").lower()
+	if choice == "y":
+		packages = "plasma-meta kde-applications-meta plasma-wayland-session sddm"
```

Comment:
Per my comments and what Torxed said on the combine KDE issue, can we change this from a choice to a notice? Instead of conditional installation of these packages, let's install them in all cases and in the `if "nvidia" in _gfx_driver_packages:` conditional, print "Plasma has known compatibility issues with the proprietary Nvidia driver. Using the Xorg session with this driver is recommended." and set xorg as default.

Reply:
Plus, this is wrong as the only package just for the wayland session is plasma-wayland-session. The rest are installed regardless.